swagger-spec/institutions/nodes_list.yaml

```diff
@@ -1 +1,168 @@
 # /institutions/{institution_id}/nodes/
+get:
+  summary: List all institution affiliated nodes
```

Comment:
Since this endpoint returns information related to a single institution, I think "Retrieve nodes affiliated with an institution" might be a more consistent summary.